Makefile

```diff
@@ -144,3 +145,6 @@ stop-prod:
 .PHONY: environ-regen
 environ-regen:
 	bin/mk-environ-file.sh
+
+mjml:
```

Comment:
@cleac long story short: normal target in makefile is some file or directory which is expected to be created by that target; if such resource exists execution of such target does nothing. Marking the target as .phony instructs make to skip such check and run the target command list any way.